.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": "explicit"
+  },
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "eslint.validate": ["javascript", "typescript", "html"],
+  "editor.formatOnSave": true,
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "eslint.nodePath": "node_modules",
+  "prettier.prettierPath": "./node_modules/prettier",
```

Comment:
Are you sure these two paths are needed? I'd think that it would default to looking for the eslint and prettier executables in `node_modules`.

Reply:
Yes, we need them, otherwise when I save a file, I get these prettier errors

```
"ERROR" - 4:21:49 PM] this.prettierModule.resolveConfigFile is not a function
TypeError: this.prettierModule.resolveConfigFile is not a function
```